src/sortStudents.ts

```diff
@@ -1,16 +1,95 @@
-
 export interface Student {
-  // describe Student interface
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name,
+  Surname,
+  Age,
+  Married,
+  AverageGrade
 }
 
-// create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
+
+export function sortStudents(
+  students: Student[],
+  sortBy: SortType,
+  order: SortOrder,
+): Student[] {
+  const copyStudents = students.map((el: Student) => ({ ...el }));
+
+  switch (sortBy) {
+    case SortType.Name:
+      return (copyStudents
+        .sort((
+          student1: Student,
+          student2: Student,
+        ) => {
+          return order === 'asc'
+            ? student1.name.localeCompare(student2.name)
+            : +!student1.name.localeCompare(student2.name);
```

Comment:
```suggestion
            : Number(!student1.name.localeCompare(student2.name));
```
Change like this in whole code :)